website/source/docs/platform/k8s/helm.html.md

```diff
@@ -122,6 +122,18 @@ and consider if they're appropriate for your deployment.
   used when bootstrapping new clusters, it has no effect during ongoing cluster
   maintenance.
 
+  - <a name="v-server-storage" href="#v-server-storage">`storage`</a> (`string: 10Gi`) -
+  This defines the disk size for configuring the servers' StatefulSet storage. This
+  should be set to the disk size of the attached volume.
```

Comment:
Don't we _create_ the volume at least in some cases? If so that last sentence is a little confusing - maybe we could clarify the behaviour/applicability both for existing volumes and ones we are creating?

Reply:
This is a good call out. I've added information about both the dynamic and manually created uses.